src/hb-ot-stat-table.hh

```diff
@@ -230,18 +252,58 @@ struct STAT
   bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
-    return_trace (likely (c->check_struct (this) &&
-			  majorVersion == 1 &&
-			  minorVersion > 0 &&
+    return_trace (likely (version.sanitize (c) &&
+                          c->check_struct (this) &&
+			  version.major == 1 &&
```

Comment:
This change is very wrong.  Please rewrite it to be equivalent to existing check, or otherwise make it match the spec.  As it is, it's just wrong.